components/git/staging.js

```diff
@@ -0,0 +1,67 @@
+import CLI from '../../lib/cli.js';
+import { runPromise } from '../../lib/run.js';
+import { Staging } from '../../lib/staging.js';
+
+export const command = 'staging';
+export const describe = 'Automatic port commits to a release line branch';
+
+const stagingOptions = {
+  backport: {
+    describe: 'The PR ID / number to backport, skip staging commits',
+    type: 'number'
+  },
+  paginate: {
+    describe: 'Sets a maximum number of commits to port',
+    type: 'number'
+  },
+  skipGH: {
```

Comment:
We might want to include a `labelOnly` flag if we decide to open a single issue with a list of commits with a necessary manual backport.

Reply:
I don't think we'll need it seems to be a consensus that we don't want to automatically spam all the PRs with comments.